butane_core/src/migrations/fsmigrations.rs

```diff
@@ -68,11 +68,14 @@ impl FsMigration {
     }
 
     fn write_info(&self, info: &MigrationInfo) -> Result<()> {
-        self.write_contents("info.json", serde_json::to_string(info)?.as_bytes())
+        self.write_contents(
+            "info.json",
+            &[serde_json::to_string_pretty(info)?.as_bytes(), b"\n"].concat(),
```

Comment:
[nit] I doubt this is a terribly performance-sensitive path, and I haven't benchmarked, but my guess is that all these concats are slower than doing multiple writes on the same open file.